packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -505,7 +525,16 @@ export class MatchingDataAttestationGroup {
   getAttestationsForBlock(fork: ForkName, notSeenAttestingIndices: Set<number>): AttestationNonParticipant[] {
     const attestations: AttestationNonParticipant[] = [];
     const forkSeq = ForkSeq[fork];
+    const isAfterElectra = forkSeq >= ForkSeq.electra;
```

Comment:
all of these should likley be renamed for consistency

```suggestion
    const isPostElectra = forkSeq >= ForkSeq.electra;
```